server/xpub/entities/manuscript/resolvers.js

```diff
@@ -164,52 +164,43 @@ ${err}`,
       )
 
       const pubsub = await getPubsub()
-      let uploadedSize = 0
-      let startTime = process.hrtime()
-      const progressReport = new Transform({
-        transform(chunk, encoding, done) {
+      const reportProgress = lodash.throttle(progress => {
+        pubsub.publish(`${ON_UPLOAD_PROGRESS}.${user}`, {
+          uploadProgress: Math.floor(progress * 100),
+        })
+      }, 200)
```

Comment:
Every 200ms now?

Reply:
Oh, sorry. I was trying different values when it was getting stuck due to the streams being full and forgot to change back. Should I revert to 500?

Reply:
Either that or comment here :)
https://github.com/elifesciences/elife-xpub/issues/650